tests/test_input_model.py

```
@@ -1140,3 +1144,619 @@ def test_input_model_config_class(tmp_path: Path) -> None:
+    output_path = tmp_path / "output.py"
+    with freeze_time(TIMESTAMP):
+        return_code = main(["--output", str(output_path)])
+    assert return_code == Exit.OK
+    assert_output(
+        output_path.read_text(encoding="utf-8"),
+        EXPECTED_INPUT_MODEL_PATH / "no_inheritance.py",
+    )
+
+
+@SKIP_PYDANTIC_V1
+def test_input_model_output_model_type_default(tmp_path: Path) -> None:
```

Ruff (0.14.10)

Unused function argument: `tmp_path`

(ARG001)